lib/services.rb

```diff
@@ -27,12 +27,13 @@
 Services.register(:ht_organizations) { DataSources::HTOrganizations.new }
 Services.register(:ht_collections) { DataSources::HTCollections.new }
 Services.register(:logger) do
-  Logger.new($stderr, level: Logger::INFO)
```

Comment:
Not sure why we chose to log to stderr, but best practice these days for k8s is to log to stdout; I've tripped across this a couple times in trying to redirect / review log output locally.